app/classifier/tasks/survey/choice.cjsx

```diff
@@ -44,15 +44,19 @@ ImageFlipper = React.createClass
 module.exports = React.createClass
   displayName: 'Choice'
 
+  propTypes:
+    annotation: React.PropTypes.object.isRequired
+
   getDefaultProps: ->
+    annotation: { answers: {} }
     task: null
     choiceID: ''
     onSwitch: Function.prototype
     onConfirm: Function.prototype
     onCancel: Function.prototype
 
   getInitialState: ->
-    answers: {}
+    answers: @props.annotation?.answers || {}
```

Comment:
Now that you've set a default value for the prop, this line becomes `answers: @props.annotation.answers`.